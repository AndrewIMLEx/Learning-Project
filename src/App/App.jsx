import React from 'react';

const App = () => <div>Hello World! Its me coming </div>;

export default App;
