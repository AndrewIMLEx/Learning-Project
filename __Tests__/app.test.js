import TestRenderer from 'react-test-renderer';
import React from 'react';

import App from './App';

it('renders a correctly', () => {
  const tree = TestRenderer.create(<App/>).toJSON();
  expect(tree).toMatchSnapshot();
});


