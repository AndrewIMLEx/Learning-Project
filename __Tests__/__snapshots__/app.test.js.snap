// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`landing page renders the landing page 1`] = `
<div>
  <div
    className="card"
  >
    <div
      className="header"
    >
      <h1>
        Guess The Number
      </h1>
      <h2>
        Non Logged In Gameplay
      </h2>
    </div>
    <div
      className="levels"
    >
      <div
        className="levelselect"
      >
        <h4>
          Select Difficulty Level
        </h4>
        <button
          id="standard"
          type="submit"
        >
          Standard
        </button>
        <button
          id="difficult"
          type="submit"
        >
          Difficult
        </button>
      </div>
      <div
        className="levelCard"
      >
        <h3
          style={
            {
              "color": "black",
            }
          }
        >
          Selected Level
        </h3>
        <p
          style={
            {
              "color": "red",
            }
          }
        >
          STANDARD
        </p>
        <form
          id="LevelForm"
        >
          <input
            id="Levelsinput"
          />
        </form>
         
        <p
          style={
            {
              "color": "red",
            }
          }
        >
          CORRECT GUESS
        </p>
        <button
          id="guess"
          type="submit"
        >
          Guess
        </button>
         
        <div
          className="playButtons"
        >
          <button
            id="play"
            type="submit"
          >
            Play Again
          </button>
          <button
            id="login"
            type="submit"
          >
            Log In
          </button>
        </div>
      </div>
    </div>
    <div
      className="tally"
    >
      <h4
        style={
          {
            "marginLeft": "4em",
          }
        }
      >
        Game Tally
      </h4>
      <form
        id="LevelForm"
      >
        <span>
          <label
            for="won"
            id="won-label"
          >
            Games Won
          </label>
          <input
            id="Tally"
          />
        </span>
        <br />
        <br />
        <span>
          <label
            for="lost"
            id="won-label"
          >
            Games Lost
          </label>
          <input
            id="Tally"
          />
        </span>
      </form>
      <form
        id="Gamesplayed"
      >
        <label
          for="gamesplayed"
          id="played"
          style={
            {
              "marginLeft": "2em",
            }
          }
        >
          Total Games Played
        </label>
        <input
          id="plays"
        />
      </form>
    </div>
    <div
      className="rules-card"
    >
      <div
        className="rules"
      >
        <h4>
          How to Play
        </h4>
        <ol>
          <li>
            Choose Difficulty Level-Easy(1-10) or Hard(1-100).
          </li>
          <li>
            The Computer will generate a random number between 1-10 for standard and 1-100 for Difficult.
          </li>
          <li>
            You only have 
            <a
              style={
                {
                  "color": "red",
                }
              }
            >
              3 chances
            </a>
             to guess a number and beat the Computer.
          </li>
          <li>
            Don't worry the computer will notify you if your guess is higher or lower than the random generated number.
          </li>
        </ol>
      </div>
    </div>
  </div>
   
</div>
`;
